src/storage/config.rs

```diff
@@ -16,7 +16,7 @@ pub const DEFAULT_DATA_DIR: &str = "./";
 pub const DEFAULT_ROCKSDB_SUB_DIR: &str = "db";
 const DEFAULT_GC_RATIO_THRESHOLD: f64 = 1.1;
 const DEFAULT_MAX_KEY_SIZE: usize = 4 * 1024;
-const DEFAULT_SCHED_CONCURRENCY: usize = 2048000;
+const DEFAULT_SCHED_CONCURRENCY: usize = 1024 * 512;
```

Comment:
Please limit the size in config, because some existing cluster already used a 100x size, it will cause OOM when apply to this new mechanism.